google/cloud/storage/internal/curl_request_builder.cc

```diff
@@ -95,6 +103,13 @@ std::string CurlRequestBuilder::UserAgentSuffix() const {
   return user_agent_suffix;
 }
 
+void CurlRequestBuilder::ValidateBuilderState(char const* where) const {
+  if (not static_cast<bool>(handle_.handle_)) {
```

Comment:
I think, given that there are two things named `handle_` here, it may help to be more explicit that the second one is a unique pointer, and not an object.  Can you instead write this?

```
if (handle_.handle_.get() == nullptr) {
```

...as also written here: https://github.com/GoogleCloudPlatform/google-cloud-cpp/blob/65269d5b3b271f4c00509be2d037b92f318d6234/google/cloud/storage/internal/curl_handle.cc#L86

Reply:
Done.